3rdparty/libconfig/scanctx.c

```diff
@@ -114,7 +114,17 @@ FILE *scanctx_push_include(struct scan_context *ctx, void *buffer,
     strcat(full_file, file);
   }
 
-  fp = fopen(full_file ? full_file : file, "rt");
+  if(full_file)
```

Comment:
space after if please.

Reply:
also full_file != NULL since it's not a bool

Reply:
all is related to hercules style, but libconfig is not hercules